lib/bambora/v1/profile_resource.rb

```diff
@@ -3,15 +3,72 @@
 module Bambora
   module V1
     class ProfileResource
-      def initialize(client:, sub_path:)
+      # Instantiate an interface to make requests against Bambora's Profiles API.
+      #
+      # @example
+      #
+      #   client = Bambora::JSONClient(base_url: '...', api_key: '...', merchant_id: '...')
+      #   profiles = Bambora::V1::ProfileResource(client: client)
+      #
+      #   # Start making requests ...
+      #
+      # @param client [Bambora::Client] An instance of Bambora::JSONClient, used to make network requests.
+      def initialize(client:)
         @client = client
-        @sub_path = sub_path
+        @sub_path = '/v1/profiles'
```

Comment:
Hard-coded value, because this is known beforehand. Reducing the overhead for consumer to get started, following the Principle of Least Astonishment. This is the main change.